app/controllers/teachers/classrooms/surveys_controller.rb

```diff
@@ -0,0 +1,28 @@
+module Teachers
+  module Classrooms
+    class SurveysController < BaseController
+      include ActionView::Helpers::TextHelper
```

Comment:
Is this line necessary? It doesn't look like you're using any `TextHelper` methods in this controller (http://api.rubyonrails.org/classes/ActionView/Helpers/TextHelper.html).